code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -325,7 +325,7 @@
 #define issyrinormusc(A)	(istype(A,/datum/reagent/medicine/C2/syriniver) || istype(A,/datum/reagent/medicine/C2/musiver)) //musc is metab of syrin so let's make sure we're not purging either
 
 /datum/reagent/medicine/C2/syriniver //Inject >> SYRINge
-	name = "Syriniver"
+	name = "Syriniver" //Same here.
 	description = "A potent antidote for intravenous use with a narrow therapeutic index, it is considered an active prodrug of musiver."
```

Comment:
change this description to reference babitol instead of musiver

Reply:
adulitol